test.sh

```diff
@@ -8,17 +8,11 @@
 set -eu
 
 VERSIONS=(
-  2.14.1
-  2.13
-  2.12
-  2.11
-  2.10
-  2.9
-  2.8
-  2.7
-  2.6
-  2.5
-  2.4
+  5.0
+  5.1
+  5.2
+  5.3
+  5.4
```

Comment:
You _could_ add 5.4.1 here, seeing as that's the one we're planning on using :wink:.